augur/data/schema-export-v2.json

```
@@ -51,44 +51,7 @@
                     }
                 },
                 "genome_annotations": {
-                    "description": "Genome annotations (e.g. genes), relative to the reference genome",
-                    "$comment": "Required for the entropy panel",
-                    "type": "object",
-                    "required": ["nuc"],
-                    "additionalProperties": false,
-                    "properties": {
-                        "nuc": {
-                            "type": "object",
-                            "properties": {
-                                "seqid":{
-                                    "description": "Sequence on which the coordinates below are valid. Could be viral segment, bacterial contig, etc",
-                                    "$comment": "currently unused by Auspice",
-                                    "type": "string"
-                                },
-                                "type": {
-                                    "description": "Type of the feature. could be mRNA, CDS, or similar",
-                                    "$comment": "currently unused by Auspice",
-                                    "type": "string"
-                                },
-                                "start": {
-                                    "description": "Gene start position (one-based, following GFF format)",
-                                    "type": "number"
-                                },
-                                "end": {
-                                    "description": "Gene end position (one-based closed, last position of feature, following GFF format)",
-                                    "type": "number"
-                                },
-                                "strand": {
-                                    "description": "Positive or negative strand",
-                                    "type": "string",
-                                    "enum": ["-","+"]
-                                }
-                            }
-                        }
-                    },
-                    "patternProperties": {
-                        "^[a-zA-Z0-9*_-]+$": {"$ref": "#/properties/meta/properties/genome_annotations/properties/nuc"}
-                    }
+                    "$ref": "genome_annotations#"
```

I think we should ensure the schemas are usable outside of Augur, so I'd prefer to rely on standard reference resolution with working URLs, e.g.

```suggestion
                    "$ref": "https://nextstrain.org/schemas/augur/annotations"
```

and then go add that URL to the existing routes supported by nextstrain.org.

---

Done